packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/textFeatures.ts

```diff
@@ -111,7 +111,7 @@ const OutdentWhenTabText: BuildInEditFeature<PluginKeyboardEvent> = {
 };
 
 /**
- * Requires @see ExperimentalFeatures.AutoHyphen to be enabled
+ * @deprecated
```

Comment:
it is ok to delete this object since we have never exported it.

Reply:
But remove it would be a breaking change?

Reply:
To keep it not breaking, you just need to keep all published things not changed. For this one, it is not public. So we can remove it, and just create a dummy feature and assign to TextFeatures and make it does nothing. For example:

const dummyFeature = {
  keys: [],
  shouldHandle: () => false,
 ...
}